File: JAVA/TAP_Unidad03_Practica03/src/Models/Carrera.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Models;

/**
 *
 * @author Carlos J
 */
public class Carrera {
    private int id;
    private String nombreCarrera;

    // Constructor
    public Carrera() { }

    // Constructor con parametros
    public Carrera(int id, String nombreCarrera)
    {
        this.id = id;
        this.nombreCarrera = nombreCarrera;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombreCarrera() {
        return nombreCarrera;
    }

    public void setNombreCarrera(String nombreCarrera) {
        this.nombreCarrera = nombreCarrera;
    }
}

File: C#/TAP_Unidad03_Practica03/TAP_Unidad03_Practica03/Models/Carrera.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TAP_Unidad03_Practica03.Models
{
    class Carrera
    {
        //Id
        private int id;
        public int Id
        {
            get { return id; }
            set { id = value; }
        }

        //Nombre de la Carrera
        private String nombreCarrera;
        public String NombreCarrera
        {
            get { return nombreCarrera; }
            set { nombreCarrera = value; }
        }

        // Constructor
        public Carrera() { }

        // Constructor con parametros
        public Carrera(int id, String nombreCarrera)
        {
            this.id = id;
            this.nombreCarrera = nombreCarrera;
        }
    }
}
